mteb/leaderboard/app.py

```diff
@@ -5,7 +5,6 @@
 from pathlib import Path
 
 import gradio as gr
-import pandas as pd
```

Comment:
I will approve this now, but we should probably not include changes of other things in PRs that are unrelated.